decisions/Gondor.txt

```diff
@@ -1660,7 +1660,7 @@ political_decisions = {
 			HRD = {
 				country_event = 2509
 				government = fiefdom
-				
+				political_reform = good_alignment
```

Comment:
What's up with this addition? When I take the decision, the alignment is changed via event 2509:
![image](https://github.com/The-Third-Age/TTA-Victoria-2/assets/33076111/ecf96b80-368a-4d0a-9b7a-cf4f61e9dc19)
![image](https://github.com/The-Third-Age/TTA-Victoria-2/assets/33076111/ed5807bd-e170-4888-877a-41f49c4cd9e5)

Reply:
It literally didn't work when I did it in a relatively recent game? Or was it not there yet?